airbyte-webapp/src/components/GeographyDropdown/GeographyDropdown.tsx

```diff
@@ -0,0 +1,64 @@
+import { faPlus } from "@fortawesome/free-solid-svg-icons";
+import { FontAwesomeIcon } from "@fortawesome/react-fontawesome";
+import { EU, US } from "country-flag-icons/react/3x2";
+import React from "react";
+import { components, OptionProps, MenuListProps } from "react-select";
+
+import { DropDown } from "components/ui/DropDown";
+
+import styles from "./GeographyDropdown.module.scss";
+
+const options = [
+  {
+    value: "us",
+    label: "us",
+  },
+  {
+    value: "auto",
+    label: "auto",
+  },
+  {
+    value: "eu",
+    label: "eu",
+  },
+];
+
+const flags: Record<string, React.ReactNode> = {
+  auto: <US />,
+  us: <US />,
+  eu: <EU />,
+};
+
+const GeographyOption: React.FC<OptionProps> = (props) => {
+  return (
+    <components.Option className={styles.option} {...props}>
+      <>
+        <span className={styles.flag}>{flags[props.label]}</span>
+        <span className={styles.label}>{props.label}</span>
```

Comment:
Here we will not want to output `{props.label}` but rather an internationalized string, using `props.label` as the key. Something like:

```jsx
const { formatMessage } = useIntl();

render (
...
<span className={styles.label}>{formatMessage({ id: props.label })}</span>
...
)
```

You'll have to add a corresponding entry in the `en.json` locale file too!